spec/requests/vms_spec.rb

```diff
@@ -39,6 +39,25 @@ def update_raw_power_state(state, *vms)
     end
   end
 
+  context 'Vm index' do
+    # Preload Vm recordss
```

Comment:
typo `recordss` -> `records`

Reply:
But two `s`s makes it cool!

Reply:
you can leave off the second s for "savings"